dali/operators/image/color/color_twist.h

```diff
@@ -156,6 +147,13 @@ class ColorTwistBase : public Operator<Backend> {
     }
   }
 
+  void KMagResize(int num_threads, int batch_size) {
```

Comment:
After all maybe we can make this some kind of utility as this pattern appears in more than one place.

Reply:
@mzient ?

Reply:
It's not always 1, 1. Neither is it always num_threads, batch_size for CPU - and if it is, then it should be `(num_threads, max_batch_size_)`.